build-tools/mono-runtimes/mono-runtimes.props

```diff
@@ -27,6 +27,12 @@
     <!-- LLVM -->
     <_LlvmNeeded Condition=" '$(AndroidSupportedTargetAotAbis)' != '' ">yes</_LlvmNeeded>
     <_LlvmCanBuild64 Condition=" '$(HostBits)' == '64' ">yes</_LlvmCanBuild64>
+    <_LlvmNeededWindows32></_LlvmNeededWindows32>
```

Comment:
That won't override values set on the command-line. `msbuild /p:_LlvmNeededWindows32=Garbage` *always* takes precedence, *and* is **immutable**.